lib/bolt/executor.rb

```diff
@@ -165,20 +165,32 @@ def log_plan(plan_name)
 
     def report_transport(transport, count)
       name = transport.class.name.split('::').last.downcase
-      @analytics&.event('Transport', 'initialize', name, count) unless @reported_transports.include?(name)
+      unless @reported_transports.include?(name)
+        @analytics&.event('Transport', 'initialize', label: name, value: count)
+      end
       @reported_transports.add(name)
     end
 
     def report_function_call(function)
-      @analytics&.event('Plan', 'call_function', function)
+      @analytics&.event('Plan', 'call_function', label: function)
     end
 
     def report_bundled_content(mode, name)
       if @bundled_content&.include?(name)
-        @analytics&.event('Bundled Content', mode, name)
+        @analytics&.event('Bundled Content', mode, label: name)
       end
     end
 
+    def report_apply(statement_count, resource_counts)
+      sum = resource_counts.inject(0) { |accum, i| accum + i }
+      mean = sum / resource_counts.length.to_f
+      variance = resource_counts.inject(0) { |accum, i| accum + (i - mean)**2 } / resource_counts.length.to_f
+      std_dev = Math.sqrt(variance)
+
+      data = { statement_count: statement_count, resource_mean: mean, resource_stddev: std_dev }
```

Comment:
I'm not sure what std dev will really tell us, since it's difficult for us to correlate it to the mean. We could potentially send coefficient of variation instead, but it's still unclear what questions that would help us answer...